src/main.cpp

```diff
@@ -42,7 +42,7 @@
 using namespace std;
 
 
-#define VERSION "1.10-pre"
+#define VERSION "1.10rc2"

```

Comment:
Thanks for changing this. I had just cut a v1.10rc2 before I saw this change. I'll delete that release and we will re-cut rc2 to include this PR.